packages/cashier/src/pages/payment-agent/payment-agent-disclaimer/payment-agent-disclaimer.scss

```diff
@@ -0,0 +1,9 @@
+.payment-agent-disclaimer {
```

Comment:
@george-usynin-binary Is there any style guide for `Sass` where I can read to get familiar with our coding style? 🙏🏻

Reply:
sure @farzin-fs 
https://github.com/binary-com/deriv-app/blob/master/docs/Stylesheet/README.md